openshift/template.yaml

```diff
@@ -190,3 +190,9 @@ parameters:
   required: false
   name: JAVA_OPTIONS
   value: ""
+
+- description: Gremlin pool size
```

Comment:
Why is this change required? The parameter is not used anywhere.

Reply:
It is used to control the gremlinPool.
https://github.com/fabric8-analytics/gremlin-docker/blob/master/scripts/entrypoint.sh#L56

We can override from saas templates to check whether it has any effect on the gremlin query delay.

Reply:
My suggestion is to leave it as 8 as of now. Until we understand it completely lets not change it. I am fine with adding this for flexibility.